aten/src/ATen/native/mps/MPSFallback.cpp

```diff
@@ -0,0 +1,208 @@
+//  Copyright © 2022 Apple Inc.
+
+#include <ATen/ATen.h>
+#include <ATen/native/mps/Copy.h>
+#include <ATen/core/dispatch/Dispatcher.h>
+#include <ATen/core/op_registration/op_registration.h>
+#include <ATen/native/MathBitsFallback.h>
+#include <ATen/core/ivalue.h>
+#include <ATen/core/stack.h>
+#include <ATen/core/boxing/KernelFunction.h>
+#include <ATen/core/dispatch/Dispatcher.h>
+#include <torch/library.h>
+#include <ATen/Functions.h>
+
+namespace at {
+
+struct MPSFallback {
```

Comment:
Is this modified from the general CPUFallback? Looks like no comment has been updated?
If not, can we just re-use the CPU fallback directly? Otherwise, we should cleanup the comments and mention where we diverge from it.

Reply:
cc @bdhirsh

Reply:
It is modified from CPUFallback. I have updated the comments and added a `TODO` for refactoring it later so that the logic can be shared with CPUFallback.

Reply:
We have removed the MPSFallback and using the CPUFallback.